domain/src/main/kotlin/no/nav/su/se/bakover/domain/revurdering/GjeldendeVedtaksdata.kt

```diff
@@ -42,5 +45,14 @@ data class KopierGjeldendeGrunnlagsdataOgVilkårsvurderinger(
         vilkårsvurderinger = Vilkårsvurderinger(
             uføre = uføreGrunnlagOgVilkår.second,
         )
+        gjeldendePeriodeTilOriginaltVedtak = vedtakstidslinje.associate { it.periode to it.originaltVedtak }
```

Comment:
Burde vi ikke heller her validere at den er a) sammenhengende (så kan tidslinjeForVedtakErSammenhengende være sammenhengende)? Og burde vi og validere at den er sortert? Map støtter vel implementasjoner som ikke er sortert.

Reply:
Akkurat det med sammenhengende er først og fremst en begrensning som kommer til revurdering, ikke til uthenting av gjeldende vedtaksdata. Kunne selvsagt vært litt strengere og tryna dersom en slik situasjon oppsto? Burde. sannsynligvis sørge for at den er sortert kronologisk ja, enig i det.

Reply:
Trenger vi funksjonaliteten for å ha ikke-sammenhengende gjeldendevedtak nå? Frontenden gjør jo en GET på dette for å vise "eksisterende vedtaksinformasjon"

Reply:
Tenker at noe av denne jobben kan offloades til tidslinja. Den kan validere seg selv og sjekke at ting er fint og vel, i tillegg til at den bør kunne svare på hva som er gyldig for hvilke perioder/datoer.